packages/now-build-utils/test/unit.test.js

```diff
@@ -255,3 +256,13 @@ it('should support require by path for legacy builders', () => {
   expect(FileRef2).toBe(index.FileRef);
   expect(Lambda2).toBe(index.Lambda);
 });
+
+it('should have correct $PATH when running `runPackageJsonScript()` with yarn', async () => {
+  const fixture = path.join(__dirname, 'fixtures/19-yarn-v2');
```

Comment:
```suggestion
  const fixture = path.join(__dirname, 'fixtures', '19-yarn-v2');
```

Reply:
Doesn't really make a difference. Node.js normalizes the slashes, even on Windows.